ReadMe.md

```diff
@@ -6,4 +6,4 @@ First experiment
 
 Hello
 
-Sup, my name is Phil.
+Sup, my name is Phil! I am great!
```

Comment:
I really think I got a little too excited there, might be worth adjusting the tone a bit.

```suggestion
Sup, my name is Phil. I am great.
```